client/src/components/Layout/Sidebar/index.tsx

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
+import { List } from "./styles";
 
 const Sidebar = () => {
   return (
     <div>
```

Comment:
let's keep convention with naming parent styled component like StyledSidebar etc. even if there is no styles